powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/ActivationServiceBehavior.java

```diff
@@ -438,6 +441,7 @@ public GetActivationStatusResponse getActivationStatus(String activationId, Stri
                     response.setPlatform(activation.getPlatform());
                     response.setDeviceInfo(activation.getDeviceInfo());
                     response.getActivationFlags().addAll(activation.getFlags());
+                    response.getApplicationRoles().addAll(application.getRoles());
```

Comment:
🤔 This was not mentioned in the issue nor it the commit message.

Reply:
This is the result of inconsistency I fixed along the way - we return this value in list but not in detail, seemed like an oversight.